### DISCUSSION TITLE
Laravel release 10

### DISCUSSION THREAD

**Discussion by AnthoniG** [laravel/framework#46007] [Q&A] [answered]
Laravel 10 was supposed to be released today (7th Feb 2023). 
However it now says **Q1 2023**

Anyone know what's going on.

**Comment by henzeb** [comment]
The day ain't over yet.

**Comment by johnpuddephatt** [comment, marked as answer]
Taylor says it'll probably be next week. 

He's replied to this tweet: https://twitter.com/driesvints/status/1622992537488162818
